lib/legacy-table.js

```diff
@@ -5,7 +5,7 @@ var Promise    = require('ember-cli/lib/ext/promise');
 
 module.exports = CoreObject.extend({
   init: function(plugin, revisions) {
-    this._super.init && this._super.init.apply(this, arguments);
+    this._super();
```

Comment:
Thanks for this @lcpriest . Should you be passing the args through to `this._super`?